From 47ca6a0cb1c56de60483c4888a5206db4f8ec8ba Mon Sep 17 00:00:00 2001
From: Biwen Li <biwen.li@nxp.com>
Date: Wed, 4 Sep 2019 11:39:30 +0800
Subject: [PATCH 143/245] arm64: ls1028a-qds: correct bus of rtc

commit d7a34f80c9bde3900e6561a766906c0b19b9fdeb from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

The rtc is on i2c2 bus(hardware), not on i2c1 channel 3,
so correct it

Signed-off-by: Biwen Li <biwen.li@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
index 4b125f0aa196..e283e85531ed 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
@@ -187,11 +187,6 @@
 			#size-cells = <0>;
 			reg = <0x3>;
 
-			rtc@51 {
-				compatible = "nxp,pcf2129";
-				reg = <0x51>;
-			};
-
 			eeprom@56 {
 				compatible = "atmel,24c512";
 				reg = <0x56>;
@@ -246,6 +241,14 @@
 	phy-connection-type = "rgmii-id";
 };
 
+&i2c1 {
+	status = "okay";
+	rtc@51 {
+		compatible = "nxp,pcf2129";
+		reg = <0x51>;
+	};
+};
+
 &fspi {
 	status = "okay";
 	mt35xu02g: flash@0 {
-- 
2.17.1

